### ISSUE TITLE
Point cloud test model?

### ISSUE THREAD

**Issue by zeux** [KhronosGroup/glTF-Sample-Assets#31]
One of the important use cases of glTF I've seen that doesn't seem to be covered in this repository are point clouds.

glTF-Sample-Models had MeshPrimitiveTest but I'm not really suggesting including that one - it feels like we should have some test instead that is a more realistic/heavy-weight test of point clouds specifically.

Sketchfab has a lot of assets that are point cloud scans but I'm not sure what the licensing restrictions here are. I'm thinking of something along these lines: https://sketchfab.com/3d-models/flower-point-cloud-photogrammetry-13a409d10f154d5dbfd009fe75d52222

**Comment by DrX3D** [comment]
I agree (that there should be heavy point cloud models). The model you suggested (Flower Point Cloud Photogrammetry) is great because it is CC-BY. It is currently in XYZ point cloud format. Can someone convert this to glTF?

**Comment by zeux** [comment]
Sketchfab natively supports conversion to glTF; for convenience I'm attaching the glTF export here for the asset above:

[flower_point_cloud_photogrammetry.zip](https://github.com/KhronosGroup/glTF-Sample-Assets/files/12839603/flower_point_cloud_photogrammetry.zip)

**Comment by zeux** [comment]
Hmm maybe this specific export isn't ideal - in addition to position and color, the file also contains normal data but it's redundant (every point has the same normal), which is probably a mistake in source asset or in the converter. So we'll need to post-process this or find a better scan.

**Comment by zeux** [comment]
Okay, I cleaned up the glTF file manually (using glTF-Transform to repack the buffer data. Attaching the result.

gltf.report loads and renders the scene correctly, modulo what looks like sRGB issues:

![image](https://github.com/KhronosGroup/glTF-Sample-Assets/assets/1106629/99bf40e0-b1d9-48b1-983a-a588ac3eb360)

Babylon.JS:

![image](https://github.com/KhronosGroup/glTF-Sample-Assets/assets/1106629/89c7ec92-bf62-4fa8-8f7d-a0c6890e1c66)

Sketchfab for comparison:

![image](https://github.com/KhronosGroup/glTF-Sample-Assets/assets/1106629/af59c4b3-a6da-4f78-a40a-727902cde1e9)

[flower_point_cloud_cleaned.zip](https://github.com/KhronosGroup/glTF-Sample-Assets/files/12839633/flower_point_cloud_cleaned.zip)

**Comment by zeux** [comment]
Ok, yeah, I had to convert the colors from sRGB to linear color space using a one-off C program. After conversion, the Babylon.JS render is now correct (using point size=2 here); I think we should now be able to use this asset.

![image](https://github.com/KhronosGroup/glTF-Sample-Assets/assets/1106629/61a89956-e6f7-44e8-b552-e504d10b72a2)

[flower_point_cloud_linear.zip](https://github.com/KhronosGroup/glTF-Sample-Assets/files/12839653/flower_point_cloud_linear.zip)

**Comment by wallabyway** [comment]
Compressed with MeshOpt and Unlit extension

https://github.com/wallabyway/minimal-pointcloud-gltf#flower
